extras/scripts/disable-multipath.sh

```diff
@@ -1,11 +1,46 @@
 #!/usr/bin/env bash
+#
+# stop and disable multipathd (the daemon) *and* blacklist all device names like 
+# /dev/sda so they do not appear in the output of multipath (the CLI)
+#
 
+# shell exit with error (errexit) if any command returns exit != 0 or any variable
+# is undefined (nounset)
+set -ue
+
+warn(){
+echo <<\EOF >&2
```

Comment:
Ken did you mean to use cat here?

cat <<\EOF >&2
this script will prevent listing iscsi devices when multipath CLI is called
without parameters, and so no LUNs will be discovered by applications like VDSM
(oVirt, RHV) which shell-out to call `/usr/sbin/multipath` after target login
EOF
}

Because if you use echo and here documents along with it. It doesn't print the message when warn is called.